.circleci/config.yml

```diff
@@ -147,7 +147,7 @@ jobs:
       - run: make docker.prebuilt
       - run: bin/testEnvRootMinikube.sh wait
       - run: docker images
-      - run: $GOPATH/bin/pilot-integration-test --logtostderr -hub $HUB -tag $TAG -mixer=false -auth=enable -errorlogsdir=/home/circleci/logs -use-initializer
+      - run: $GOPATH/bin/pilot-integration-test --logtostderr -hub $HUB -tag $TAG -mixer=true -auth=enable -errorlogsdir=/home/circleci/logs -use-initializer
```

Comment:
just remove `-mixer` argument

Reply:
No please keep it. It helps debug things later.. and one does not have to dig through the code to debug.

Reply:
`pilot-integration-test --help` is your friend